aws-lambda-layer/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -x
+
+set -e
+
+FREETDS_VERSION=${FREETDS_VERSION:=1.1.6}
+IMAGE_NAME=veracross/freetds-layer:${FREETDS_VERSION:latest}
```

Comment:
`veracross/freetds`

Reply:
Actually, how do we want to tag these image?

`veracross/freetds:aws-<freetds-version>`?